seed/views/v3/postoffice.py

```diff
@@ -0,0 +1,70 @@
+# !/usr/bin/env python
+# encoding: utf-8
+"""
+:copyright (c) 2014 - 2020, The Regents of the University of California,
+through Lawrence Berkeley National Laboratory (subject to receipt of any
+required approvals from the U.S. Department of Energy) and contributors.
+All rights reserved.  # NOQA
+
+"""
+from django.conf import settings
+from django.forms.models import model_to_dict
+
+from seed.serializers.postoffice import PostOfficeSerializer, PostOfficeEmailSerializer
+from seed.utils.viewsets import SEEDOrgModelViewSet
+from seed.models import PropertyState, TaxLotState
+from seed.models import PostOfficeEmail, PostOfficeEmailTemplate
+from post_office import mail
+
+
+class PostOfficeViewSet(SEEDOrgModelViewSet):
+    model = PostOfficeEmailTemplate
+    serializer_class = PostOfficeSerializer
+    pagination_class = None
+
+    def get_queryset(self):
+        return PostOfficeEmailTemplate.objects.filter(
+            organization_id=self.get_organization(self.request)
+        ).order_by('name')
+
+    def perform_create(self, serializer):
+        org_id = self.get_organization(self.request)
+        user = self.request.user
+        serializer.save(organization_id=org_id, user=user)
+
+
+class PostOfficeEmailViewSet(SEEDOrgModelViewSet):
+    model = PostOfficeEmail
+    serializer_class = PostOfficeEmailSerializer
+    pagination_class = None
+
+    def get_queryset(self):
+        return PostOfficeEmail.objects.all()
+
+    def perform_create(self, serializer):
+        template_id = self.request.data.get('template_id')
+        inventory_id = self.request.data.get('inventory_id', [])
+        if self.request.data.get('inventory_type') == "properties":
+            state = PropertyState
+        else:
+            state = TaxLotState
+        properties = state.objects.filter(id__in=inventory_id)
```

Comment:
I think we might need add org id as a filter param here. As is, I think a user could pass IDs of states that aren't part of their org.

This wouldn't necessarily leak leak data (the email is still going to the property owner), but should probably be avoided.